explainaboard/loaders/file_loader.py

```diff
@@ -306,12 +484,41 @@ def replace_labels(cls, features: dict, example: dict) -> dict:
                 new_example[examp_k] = examp_v
         return new_example
 
-    def load_raw(self, data: str | DatalabLoaderOption, source: Source) -> list[dict]:
+    def load_raw(
+        self, data: str | DatalabLoaderOption, source: Source
+    ) -> FileLoaderReturn:
         config = narrow(data, DatalabLoaderOption)
         dataset = load_dataset(
             config.dataset, config.subdataset, split=config.split, streaming=False
         )
-        return [self.replace_labels(dataset.info.features, x) for x in dataset]
+        # TODO(gneubig): patch for an inconsistency in datalab, where DatasetDict
+        #  doesn't have info
+        if isinstance(dataset, DatasetDict) or isinstance(dataset, IterableDatasetDict):
+            raise ValueError('Cannot handle DatasetDict returns')
+        info = dataset.info
```

Comment:
Have we added `info` as a member variable into the `DatasetDict` class in DataLab (we have `DatasetDict[split]._info`)? Sometime it should be split-dependent. For example, features of test samples are not necessarily the same as features of training samples.

Reply:
I don't think this exists as a member variable in DatasetDict (it threw a `mypy` error). It probably should be fixed in DataLab.

Reply:
yeah, by design, there is no definition of `info` for DatasetDict but there is for `Dataset`. But we can introduce `info` for DatasetDict as well,  just highlighting a caveat as above: `Sometime it should be split-dependent. For example, features of test samples are not necessarily the same as features of training samples.`